src/main/java/org/folio/service/uc/UcCostPerUseServiceImpl.java

```diff
@@ -316,13 +337,17 @@ private CompletableFuture<Map<String, UcCostAnalysis>> fetchTitlePackagesCost(
       .map(cr -> new UcTitlePackageId(cr.getTitleId(), cr.getPackageId()))
       .distinct()
       .collect(Collectors.toList());
+    log.debug("fetchTitlePackagesCost:: Fetching Title package cost with ids: {}",
+      titlePackageIds.stream().map(UcTitlePackageId::toString).collect(Collectors.joining(",")));
     var configuration = createGetTitlePackageConfiguration(ucConfiguration, true);
     return loadFromCache(titlePackageIds, configuration);
   }
 
   private CompletableFuture<Map<String, UcCostAnalysis>> fetchTitlePackageCost(List<UcTitlePackageId> titlePackageIds,
                                                                                PlatformType platformType,
                                                                                CommonUcConfiguration ucConfiguration) {
+    log.debug("fetchTitlePackageCost:: Fetching Title Package Cost for platformType: {} with ids: {}",
```

Comment:
Why this method starts with `debug` log while other start with `info` level?

Reply:
changed.